experiments/dataSources/SalesforceREST/readme.md

```diff
@@ -0,0 +1,54 @@
+# Show MORE Salesforce data in Skuid using REST
+The standard Salesforce datasource allows you to connect to most sObjects and build compelling UI's with that data.  However there are some types of data within your org not accessible through the standard Salesforce Datasource.  Boo....
+
+Don't worry! Salesforce provides a rich array of APIs that provide access to this type of data. In Skuid, it's easy to create a REST datasource to retrieve data from these API's and create custom UI's.
+
+Examples of this data include: 
+
+* Access to [Data.com search](https://developer.salesforce.com/docs/atlas.en-us.datadotcom_api_dev_guide.meta/datadotcom_api_dev_guide/datadotcom_api_dev_guide_intro.htm)
+* Access to the [interface API](https://developer.salesforce.com/docs/atlas.en-us.uiapi.meta/uiapi/ui_api_get_started.htm)-  like creating a model of field picklist values. 
+* Access to many [setup objects](https://developer.salesforce.com/docs/atlas.en-us.220.0.object_reference.meta/object_reference/sforce_api_objects_concepts.htm) not exposed as objects to Skuid.  
+
+
+## How to use this data source
+
+### Create the data source
```

Comment:
We don't offer a specific name in this README. In your sample pages it's `SFDC_API`. I think we should add that as an example here, since users will be prompted to select the appropriate data source when adding your pages.